mep/books/models.py

```diff
@@ -313,6 +313,19 @@ class Work(TrackChangesModel, Notable, ModelIndexable, EventSetMixin):
     genres = models.ManyToManyField(
         Genre, blank=True, help_text="Genre(s) from OCLC record"
     )
+
+    # top level category
+    category = models.ForeignKey(
+        Genre,
```

Comment:
just to make sure I'm clear: we're not restricting which genres can be selected as top-level categories, but just relying on the content admins to use it sensibly, right?

(I feel like this merits a comment or documentation of some kind somewhere)

Reply:
Correct